documentation/plain_rules.md

```diff
@@ -119,7 +131,7 @@ Updates an specified attribute of a given entity (in the Context Broker instance
 * attrType: optional, type of the attribute to set. By default, not set (in which case, only the attribute value is changed).
 * trust: optional, trust token for getting an access token from Auth Server which can be used to get to a Context Broker behind a PEP.
 
-The values of these fields can be either literal values or use `${X}` substitution macros (except trust), where `X` may be:
+The values of these fields can be either literal values or use `${X}` attribute substitution (except trust), where `X` may be:
```

Comment:
This (and the 3 bullets below) is the part that should be removed (as it is already explsined in the "attributes substuttion" section) or replaced to a reference to that section.